src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+
+**As the crew:** Maybe you’ve always wanted to beat your boss senseless. Now you can! But for the strongly-loyal crew, running the station without getting turned is a game of risk-versus-reward. After all, Nanotransen isn’t going to just feed you into a wall of bullets and lasers, but the Head Revs totally might. Pity the medbay; if the round turns into a storming of the palace scenario, they are suddenly going to be very busy.
+
+## Why this is fun for the admins
+
+Revs is a mode where things ramp up in drama over the course of the round, but the regular operation of the station probably isn’t directly at risk. Both sides have specific win conditions (the elimination of the other), with the station being pulled between them. This means it’s different from the normal violence-and-gibbing approach that traitors and operatives cause. This also means no more having multiple layers of anti-metagame legislation to control round flow.
+
+## Important Refactors and New Content
+
+### Head Revolutionary abilities are tied to a Revolutionary Implant.
+
+This implant does three things:
+
+1. Gives the HR innate access to talk on the Syndicate radio.
+2. Grants them a Influence conversion aura around themselves.
+3. Grants them a conversion ability that allows them to covertly flip people to Rev.
+
+Conversion works as follows:
+
+1. The Revolutionary Implant grants the owner the ability to see the Influence level of those around them.
+2. If the Head Revolutionary sees a crew member with a high enough Influence level, they can activate their implant's ability and click on the target.
+3. If successful, the target is silently converted to a Revolutionary.
+4. The ability enters a cooldown.
+
+### Removal of MindShield implants being orderable from Cargo
+
+MindShields are metagameable and make it extremely unfun to play as a Head Rev. They have no interesting interactions; they’re just a hand-wave for why some people are completely immune to a game mechanic, and whilst that’s utterly required for round flow (allowing Sec and the Heads to trust each other) it is a magic solve-everything button which leads to a total wall in design.
+
+A small number of MindShields could be kept on-station to allow staff promotions within reason. _A kind admin might decide to send a crate if the station runs out and it'd be good for the round, but this is out of scope of this design document._
+
+### Removal of Flashes being able to convert people
+
+Flashes are metagameable, confusing for newbies, hard to balance, and enforce too much reliance on manufacture by Science by the Revs for effective round flow. They also have no narrative justification beyond it being an Ur-mode from the dawn of SS13. There are people playing SS14 that are younger than the only good _Men In Black_ movie.
+
+### Addition of Influence as a tracked player stat, and associated gameplay features
```

Comment:
Is there going to be a way for a player to know their influcence level. Something vague would be ideal to keep away too much metagaming but equally you dont want to heavily impact players by something they have zero feedback on.

Reply:
Unless rev posters and randomly found about the station, having your influence change would instantly tell you what game mode is running